tests/library/test_nutrition_order.py

```diff
@@ -0,0 +1,56 @@
+import json
+import os
+import unittest
+
+
+class TestNutritionKeyOrder(unittest.TestCase):
+
+    def test_nutrition_key_order(self):
+        correct_order = [
+            "type",
```

Comment:
I think we should omit `type` - for the one website where it appears in our test data, it contains the `value` `NutritionInformation` -- that _does_ refer to the [relevant `schema.org` type](https://schema.org/NutritionInformation) -- but that's already known/inferred by the fact that we're accessing the `nutrition` key for a `Recipe`.

Reply:
I agree, I removed it from the defined list which will just push it to the bottom of the test case that expects it.